src/bindings/python/src/openvino/frontend/__init__.py

```diff
@@ -25,7 +25,6 @@
 
 # extensions
 from openvino._pyopenvino import DecoderTransformationExtension
-from openvino._pyopenvino import JsonConfigExtension
```

Comment:
should not we deprecate the extension this release? Only after we can remove it?

Reply:
It is not a public API, we can remove it

Reply:
It is not really a hidden in a way that FE API is hidden because it can be used with usual add_extension. But any way we want to remove it because there is no any documentation for it, neither samples, nor even internal usages -- nothing.